tutorials/Augurpy.ipynb

```diff
@@ -0,0 +1,195 @@
+{
```

Comment:
<p>Let's show everything that is possible here and explain briefly how it was calculated (the feature importances). We eventually have to add some plots for those as well.</p><p> </p><p>"We recommend that you try out multiple estimators depending on the data used."</p><p>If we make this statement we should also explain why one should do so and how to select the "best" among the models. Or we leave this statement out ^_^</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/augurpy/pull/49/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='9'/>

Reply:
Yess. Leaving it out until we know more. :)